test/fixtures/.lounge/config.js

```diff
@@ -3,6 +3,7 @@
 var config = require("../../../defaults/config.js");
 
 config.prefetch = true;
+config.prefetchStorage = false;
```

Comment:
Are you going to add tests for this feature or is there something blocking you from that?